# HG changeset patch
# User Rik <rik@octave.org>
# Date 1354210653 28800
#      Thu Nov 29 09:37:33 2012 -0800
# Node ID ea2788117ee389dad00540e3268ce8edbd4607ad
# Parent  9811b32b645ed61f2e60e7c34ad0d1ad171ce94c
NEWS: Add notice about extending error functions to accept complex inputs.

* NEWS: Add notice about extending error functions to accept complex inputs.

* arith.txi: Use strict alphabetical ordering for list of special functions.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -84,33 +84,42 @@ Summary of important user-visible change
 
  ** The colormap function now provides new options--"list", "register",
     and "unregister"--to list all available colormap functions, and to
     add or remove a function name from the list of known colormap
     functions.  Packages that implement extra colormaps should use these
     commands with PKG_ADD and PKG_DEL statements.
 
  ** The m-files in the image directory have been overhauled.
+
     The principal benefit is that Octave will now no longer automatically
     convert images stored with integers to doubles.  Storing images as uint8
     or uint16 requires only 1/8 or 1/4 the memory of an image stored using
     doubles.  For certain operations, such as fft2, the image must still be
     converted to double in order to work.
 
  ** The datevec function has been extended for better Matlab compatibility.
     It now accepts string inputs in the following numerical formats: 12, 21,
     22, 26, 29, 31.  This is undocumented, but verifiable, Matlab behavior.
     In addition, the default for formats which do not specify a date is
     January 1st of the current year.  The previous default was the current day,
     month, and year.  This may produce changes in existing scripts.
 
+ ** The error function and its derivatives has been extended to accept complex
+    arguments.  The following functions now accept complex inputs:
+
+    erf   erfc   erfcx   erfinv   erfcinv
+
+    In addition two new error functions erfi (imaginary error function) and
+    and dawson (scaled imaginary error function) have been added.
+
  ** The default name of the Octave crash dump file is now called
     octave-workspace instead of octave-core.
 
- ** The java package from Octave Forge is now part of Octave.  The
+ ** The java package from Octave Forge is now part of core Octave.  The
     following new functions are available for interacting with Java
     directly from Octave:
 
       java                  java_invoke
       java2mat              java_new
       javaArray             java_set
       javaMethod            java_unsigned_conversion
       javaObject            javaaddpath
@@ -124,20 +133,20 @@ Summary of important user-visible change
     are now available (provided that Octave is compiled with support for
     Java enabled):
 
       helpdlg    listdlg   questdlg
       inputdlg   msgbox    warndlg
 
  ** Other new functions added in 3.8.0:
 
-      betaincinv   erfcinv      lines      rgbplot
-      cmpermute    findfigs     polyeig    shrinkfaces
-      cmunique     fminsearch   splinefit
-      colorcube    iscolormap   tetramesh
+      betaincinv   dawson     fminsearch   rgbplot    
+      cmpermute    erfcinv    iscolormap   shrinkfaces
+      cmunique     erfi       lines        splinefit             
+      colorcube    findfigs   polyeig      tetramesh             
 
  ** Deprecated functions.
 
     The following functions were deprecated in Octave 3.4 and have been
     removed from Octave 3.8.
                                            
       autocor    dispatch              is_global    strerror
       autocov    fstat                 krylovb      values  
diff --git a/doc/interpreter/arith.txi b/doc/interpreter/arith.txi
--- a/doc/interpreter/arith.txi
+++ b/doc/interpreter/arith.txi
@@ -280,26 +280,26 @@ cos (pi/2)
 @DOCSTRING(betaln)
 
 @DOCSTRING(bincoeff)
 
 @DOCSTRING(commutation_matrix)
 
 @DOCSTRING(duplication_matrix)
 
+@DOCSTRING(dawson)
+
 @DOCSTRING(erf)
 
 @DOCSTRING(erfc)
 
 @DOCSTRING(erfcx)
 
 @DOCSTRING(erfi)
 
-@DOCSTRING(dawson)
-
 @DOCSTRING(erfinv)
 
 @DOCSTRING(erfcinv)
 
 @DOCSTRING(gamma)
 
 @DOCSTRING(gammainc)
 
